src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -138,6 +138,10 @@ public class ProgrammingExercise extends Exercise {
     @Column(name = "release_tests_with_example_solution", table = "programming_exercise_details")
     private boolean releaseTestsWithExampleSolution;
 
+    @Nullable
+    @Column(name = "iris_activated")
+    private Boolean irisActivated;
```

Comment:
As described in the changelog, this should probably not be nullable and instead use `false` as a default.